### ISSUE TITLE
[capd] failed to create capi-quickstart.yaml

### ISSUE THREAD

**Issue by gyliu513** [kubernetes-sigs/cluster-api#4673]
**What steps did you take and what happened:**
Following the guidance at https://cluster-api.sigs.k8s.io/user/quick-start.html for CAPD.

```console
root@xanthene1:~/capi# clusterctl init --infrastructure docker
Fetching providers
Installing cert-manager Version="v1.1.0"
Waiting for cert-manager to be available...
Installing Provider="cluster-api" Version="v0.3.16" TargetNamespace="capi-system"
Installing Provider="bootstrap-kubeadm" Version="v0.3.16" TargetNamespace="capi-kubeadm-bootstrap-system"
Installing Provider="control-plane-kubeadm" Version="v0.3.16" TargetNamespace="capi-kubeadm-control-plane-system"
Installing Provider="infrastructure-docker" Version="v0.3.16" TargetNamespace="capd-system"

Your management cluster has been initialized successfully!

You can now create your first workload cluster by running the following:

  clusterctl config cluster [name] --kubernetes-version [version] | kubectl apply -f -

```
```console
root@xanthene1:~/capi# # The list of service CIDR, default ["10.128.0.0/12"]
root@xanthene1:~/capi# export SERVICE_CIDR=["10.96.0.0/12"]
root@xanthene1:~/capi#
root@xanthene1:~/capi# # The list of pod CIDR, default ["192.168.0.0/16"]
root@xanthene1:~/capi# export POD_CIDR=["192.168.0.0/16"]
root@xanthene1:~/capi#
root@xanthene1:~/capi# # The service domain, default "cluster.local"
root@xanthene1:~/capi# export SERVICE_DOMAIN="k8s.test"
```
```console
root@xanthene1:~/capi# clusterctl config cluster capi-quickstart \
>   --kubernetes-version v1.18.16 \
>   --control-plane-machine-count=3 \
>   --worker-machine-count=3 \
>   > capi-quickstart.yaml

Error: failed to read "cluster-template.yaml" from provider's repository "infrastructure-docker": failed to download files from GitHub release v0.3.16: failed to get file "cluster-template.yaml" from "v0.3.16" release
```

**What did you expect to happen:**

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Environment:**

- Cluster-api version:
- Minikube/KIND version:
- Kubernetes version: (use `kubectl version`):
- OS (e.g. from `/etc/os-release`):

/kind bug
[One or more /area label. See https://github.com/kubernetes-sigs/cluster-api/labels?q=area for the list of labels]

**Comment by schrej** [comment]
Since you didn't provide a clusterctl version in your issue this is just a guess, but if you're using the latest development version, I think you'll need to use local artifacts as described here: https://cluster-api.sigs.k8s.io/clusterctl/developers.html#use-local-artifacts

**Comment by gyliu513** [comment]
```console
root@xanthene1:~/capi# clusterctl version
clusterctl version: &version.Info{Major:"0", Minor:"3", GitVersion:"v0.3.16", GitCommit:"6d7649fc5f407ac48b6d8e3929c97822527c0454", GitTreeState:"clean", BuildDate:"2021-04-16T01:01:02Z", GoVersion:"go1.13.15", Compiler:"gc", Platform:"linux/amd64"}
```

I was using 0.3.16, @schrej

**Comment by fabriziopandini** [comment]
When doing "clusterctl config cluster" with CAPD you have to add `--flavor development`(as per https://cluster-api.sigs.k8s.io/user/quick-start.html#generating-the-cluster-configuration)
Also, I got reports that kind images for v1.18.16 are not available on GitHub, so I suggest to use v1.18.19 (see https://hub.docker.com/r/kindest/node/tags for the list of available images)

**Comment by gyliu513** [comment]
it works, thanks @fabriziopandini
